java/src/main/java/org/rocksdb/RocksDB.java

```diff
@@ -439,6 +439,12 @@ protected void storeOptionsInstance(DBOptionsInterface options) {
     options_ = options;
   }
 
+  private static void checkBounds(int offset, int len, int size) {
+    if ((offset | len | (offset + len) | (size - (offset + len))) < 0) {
```

Comment:
@thisisbaozi This isn't right. Can you fix this?

Reply:
Ok

Reply:
@sagar0 I tested the method in various of params, it works well. Could you give me a hint?